### DISCUSSION TITLE
KB5028254 - Latest cumulative update (22621.2070) listed as stable update in some countries

### DISCUSSION THREAD

**Discussion by Girofox** [valinet/ExplorerPatcher#2129] [Q&A] [answered]
So it seems the latest cumulative update KB5028254 is actually a preview update but listed as a stable update in some countries. As you can see in attached screenshot this is the case in Germany (_second entry in list_), there was only a prompt to install but this was the case with all prior cumulative updates though. The only preview update was for .NET framework but this was hidden under advanced options, optional updates.

Do i need to uninstall the latest update or will it "turn" into a regular update after some time with the correct symbols etc.?

![image](https://github.com/valinet/ExplorerPatcher/assets/43116061/38e98775-b074-4939-a0b8-e06e93e871ee)

**Comment by pyrates999** [comment]
Duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1700

**Comment by pyrates999** [comment, marked as answer]
This should be closed because it is for a previous preview update (https://support.microsoft.com/en-gb/topic/july-26-2023-kb5028254-os-build-22621-2070-preview-66161ea6-c106-43e9-bef5-8410ccbd34e8) that has been superseded by another update.
